projects/flow.hoomd.lj/README.md

```diff
@@ -1,53 +1,66 @@
-# About
+An Introductory Walk-through Using signac with HOOMD-blue
 
-The example project features the generation of a p-V phase diagram of a simulated Lennard-Jones (LJ) fluid and an ideal gas estimate.
+This example project features the generation of a p-V phase diagram of a simulated Lennard-Jones (LJ) fluid and an ideal gas estimate.
 The LJ fluid is sampled via molecular dynamics using the [HOOMD-blue particle simulation toolkit](https://glotzerlab.engin.umich.edu/hoomd-blue/).
 
+
 # Usage
 
-To execute the example workflow, follow these steps:
+1. Initialize the project with
+
+    ```
+    python src/init.py
+    ```
+
+- This creates the `workspace/`, which holds all of our `jobs`. Each `job` has its own directory, named by the job's unique `id` (something like `87c7fccdea3531da704bbae95e95e914`).
+- If you look in these directories, you'll see `signac_statepoint.json`. This is a json file that contains the statepoint parameters for that job.
+- NOTE: The job's `id` is generated specifically for the dict containing the statepoint parameters, so do not directly change the directory name or `signac_statepoint.json` directly.
+
+2. All of the operations we will be performing on our data is stored in `project.py`. An operation is a function with `job` as its only argument that signac-flow recognizes as a part of your workflow.
+    - You can tell which methods are operations because they will have the `@MyProject.operation` decorator, which tells signac that this method is to be treated as an operation.
+    - `operation`s typically have pre- and post- conditions. This is how signac knows when the operation should be run. For example, the `initialize()` function has no pre-condition because it must be run first, but has `@MyProject.post.isfile('init.gsd')` as a post-condition. This post-condition means that the operation will *not* be run if `init.gsd` exists.
```

Comment:
```suggestion
    - `operation`s typically have pre- and post- conditions. This is how signac-flow knows when the operation should be run. For example, the `initialize()` function has no pre-condition because it must be run first, but has `@MyProject.post.isfile('init.gsd')` as a post-condition. This post-condition means that the operation will *not* be run if `init.gsd` exists in the job directory.
```